test/runner.sh

```diff
@@ -57,4 +61,5 @@ ${PSQL} -U ${TEST_PGUSER} \
      -v ROLE_SUPERUSER=${TEST_ROLE_SUPERUSER} \
      -v ROLE_DEFAULT_PERM_USER=${TEST_ROLE_DEFAULT_PERM_USER} \
      -v ROLE_DEFAULT_PERM_USER_2=${TEST_ROLE_DEFAULT_PERM_USER_2} \
+     -v EXTENSION_LIB="'timescaledb-${EXT_VERSION}'" \
```

Comment:
Why is this needed? Does this bypass the loader and thus make the loader test useless?